### ISSUE TITLE
Future development?

### ISSUE THREAD

**Issue by janpio** [knocknarea/fastlane-plugin-ionic_integration#8]
Are you planning on any future development of this plugin?
Adding `screengrab` for Android?
Other Ionic/Cordova specific behaviour?

**Comment by knocknarea** [comment]
@janpio Absolutely. Would you be up to the challenge of adding android screen grab?
Also, while it says that it uses Ionic/Cordova it can also be used with any JS mobile framework that produces an Xcode project

**Comment by janpio** [comment]
I am currently fighting my way through getting screengrab to work with the Cordova Android project. It is painful. Let's see if something comes of it.

**Comment by janpio** [comment]
It's working, so I am looking into how to automate this similar to what you did. I might have some questions and create new issues for them ;)

Re the future development: Did you have any functionality in mind besides the `screengrab` support for Android? What else is needed to make fastlane work with Ionic/Cordova?

**Comment by knocknarea** [comment]
@janpio Excellent. Welcome on board :-) I would be really good to get it working with android.
I'd like to think it could be used as a one stop shop of ionic/cordova or other javascript framework working with fastlane.

**Comment by janpio** [comment]
That sure would be nice. But what else is missing that is Cordova/Ionic specific?

**Comment by brad** [comment]
@janpio Do you still have that working screengrab code tucked away somewhere? I could really use it right now

**Comment by janpio** [comment]
I still have these folders and plan to work on it in the future, but this is spread over multiple repositories that were not meant for public, so commit messages are crap (just needed 20 minutes to understand if what I am looking at _really_ is what I thought it was). So I would have to invest a day or two before anyone could actually use it, especially as the newer `cordova-android` versions have quite a different folder structure :/

If you have specific questions, feel free to ask them as issues at https://github.com/janpio/fastlane-plugin-cordova_screenshots/ (which is the planned fork/repo for these changes)

(Sorry)

**Comment by brad** [comment]
Thanks for the update @janpio
